discord/channel.py

```diff
@@ -3220,6 +3252,84 @@ def get_partial_message(self, message_id: int, /) -> PartialMessage:
         return PartialMessage(channel=self, id=message_id)
 
 
+class VoiceChannelEffectAnimation(NamedTuple):
+    """Represents an animation that can be sent to a voice channel.
+
+    .. versionadded:: 2.7
+    """
+
+    id: int
+    type: VoiceChannelEffectAnimationType
+
+
+class VoiceChannelSoundEffect(PartialSoundboardSound): ...
+
+
+class VoiceChannelEffectSendEvent:
+    """Represents the payload for an :func:`on_voice_channel_effect_send`
+
+    .. versionadded:: 2.7
+
+    Attributes
+    ----------
+    animation_type: :class:`int`
+        The type of animation that is being sent.
+    animation_id: :class:`int`
+        The ID of the animation that is being sent.
+    sound: Optional[:class:`SoundboardSound`]
+        The sound that is being sent, might be None if the effect is not a sound effect.
+    guild: :class:`Guild`
+        The guild that the sound is being sent in.
```

Comment:
Never end with a preposition
```suggestion
        The guild in which the sound is being sent.
```